lock/lock.go

```diff
@@ -314,6 +314,32 @@ func GetLockStatus(options *Options) (*dynamodb.GetItemOutput, error) {
 	return item, nil
 }
 
+// ScanTable will perform a scan operation on the indicated DynamoDB table
+// This operation is useful in certain cases, for example when we want to
+// generate a report of all currently active ref arch deployments tracked in our lock table
+func ScanTable(options *Options) (*dynamodb.ScanOutput, error) {
```

Comment:
NIT: Since this is in the `lock` library, I think we should call this `ScanLocks` and optimize the output for that.

Reply:
Makes sense to me. Done in [0be6576](https://github.com/gruntwork-io/go-commons/pull/60/commits/0be657646d4879a2c812cfb6c3ef81de4e182df7)